README.md

```diff
@@ -27,7 +27,28 @@ If you have a well phrased question that might benefit others, consider asking i
 
 ## Quick start:
 * Make sure you have golang version 1.6 or greater installed.
+* If you do not have it yet create your GOPATH and export it
+		
+		$ mkdir $HOME/work
+		$ export GOPATH=$HOME/work
+
+For more information you can check [GOPATH Documentation](https://golang.org/cmd/go/#hdr-GOPATH_environment_variable).
+
+As a long term solution please add /usr/local/go/bin to the PATH environment variable. You can do this by adding this line to your /etc/profile (for a system-wide installation) or $HOME/.profile:
+
+                export PATH=$PATH:/usr/local/go/bin
```

Comment:
change the spaces to tabs please.
do you really me .profile and not .bashrc ?
and why do we add /usr/local/go/bin ? don't we want $GOPATH/bin ?
Should we avoid the system wide comments?